sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/owaspzapwrapper/cli/OwaspZapWrapperCLI.java

```diff
@@ -44,8 +45,13 @@ private void start(String[] args) throws IOException {
             productMessagehelper.writeProductMessages(scanContext.getProductMessages());
 
         } catch (ZapWrapperRuntimeException e) {
-            LOG.error("Must exit with exit code {} because: {}.", e.getExitCode().getExitCode(), e.getMessage(), e);
-            productMessagehelper.writeProductError(e);
+            if (ZapWrapperExitCode.SCAN_JOB_CANCELLED.equals(e.getExitCode())) {
+                LOG.info(e.getMessage());
+                productMessagehelper.writeProductMessages(scanContext.getProductMessages());
+            } else {
+                LOG.error("Must exit with exit code {} because: {}.", e.getExitCode().getExitCode(), e.getMessage(), e);
```

Comment:
What do you want to say here?

```suggestion
                LOG.error("Must exit with exit code {} because: {}.", e.getExitCode().getExitCode(), e.getMessage(), e);
```